install-krew

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+#
+# Install krew
+#
+# License: Apache 2.0 - see https://github.com/kubernetes-sigs/krew/blob/master/LICENSE
+
+set -o pipefail
+set -o errexit
+
+STABLE_RELEASE="$(cat $(dirname $0)/STABLE_RELEASE)"
```

Comment:
You need to fetch the current `STABLE_RELEASE` file from remote. Users should not have to clone krew or download the source tarball for install.